DataAccessLayer/AppDbContext.cs

```diff
@@ -0,0 +1,87 @@
+﻿using EntityLayer.Entities;
+using EntityLayer.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer
+{
+    public class AppDbContext : DbContext
+    {
+
+        public AppDbContext(DbContextOptions<AppDbContext> options)
+      : base(options)
+        {
+        }
+        public DbSet<Zone> Zones { get; set; }
+        public DbSet<Asset> Assets { get; set; }
+        public DbSet<FloorMap> FloorMaps { get; set; }
+
+        public DbSet<AssetPositionHistory> AssetPositionHistory { get; set; }
+        //public DbSet<AssetZoneHistory> AssetZoneHistory { get; set; }
```

Comment:
is this comment needed?

Reply:
This code will be used after creating the AssetZoneHistory class.